src/Resources/skeleton/controller/Controller.tpl.php

```diff
@@ -2,10 +2,10 @@
 
 namespace <?= $namespace; ?>;
 
+use Symfony\Bundle\FrameworkBundle\Controller\<?= $parent_class_name; ?>;
 use Symfony\Component\Routing\Annotation\Route;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
-class <?= $class_name; ?> extends Controller
+class <?= $class_name; ?> extends <?= $parent_class_name; ?><?= "\n" ?>
```

Comment:
just wondering: is the `"\n"` necessary?

Reply:
In theory it's not and I didn't add it at first ... but the generated controller class was like this:

```php
class CoolController extends AbstractController {
  // ...
}
```

instead of this:

```php
class CoolController extends AbstractController
{
  // ...
}
```

Can any of you reproduce my wrong behavior? If yes, can you explain to me why, because I don't understand this 😅